Purpose: Make sure bootlogd findpty() returns an error value when it fails to
         find a usable pty
Authour: Rob Leslie
Fixes:   #492796
Status:  unknown

--- a/src/bootlogd.c
+++ b/src/bootlogd.c
@@ -189,7 +189,7 @@
 		}
 		if (found) break;
 	}
-	if (found < 0) return -1;
+	if (!found) return -1;
 
 	if (name) strcpy(name, tty);
 
